src/utils/bitSet.ts

```diff
@@ -61,3 +61,46 @@ export const toObject = ( bitMask: TBitSetMask = {}, value = 0 ): TBooleanObject
 
     return result;
 };
+
+/**
+ * Extract number of bits from number from given position.
+ *
+ * @example
+ * ```js
+ * const value = 7; // 0b111
+ *
+ * console.log(extractBitsFromNumber(value, 2, 1));
+ * // 3
+ * // i.e. 0b011
```

Comment:
```suggestion
 * // i.e. 0b11
```
?

Reply:
and it can be in one line

Reply:
yes, but `console.log` print only one number, but ok - fixed